lib/got-pollyfill-error.js

```diff
@@ -0,0 +1,8 @@
+module.exports = class GotPollyFillError extends Error {
```

Comment:
This feels dirty but felt like the path of least resistance without rewriting the error handling